docs/intro.asciidoc

```diff
@@ -9,11 +9,33 @@ The Elastic APM iOS Agent measures the performance of your mobile applications i
 The Elastic APM iOS Agent uses the https://github.com/open-telemetry/opentelemetry-swift[OpenTelemetry-Swift SDK].
 The agent automatically traces URLSessions and provides distributed traces annotated with device information along
 with your back-end services instrumented with Open-Telemetry.
+
 The agent also captures any custom open-telemetry traces or measurements created using the OpenTelemetry-Swift API.
 
+[float]
+[[manual-instrumentation]]
+=== How to add instrumentation
+This agent will configure the OpenTelementry-Swift `TracerProvider` & `MetricProvider`, and sets them as the global OpenTelemetry providers. They can be accessed through the OpenTelemetry SDK as follows:
+
+```
+let tracerProvider = OpenTelemetry.instance.tracerProvider
+let meterProvider = OpenTelemetry.instance.meterProvider
+```
+
+These objects can be used to acquire new tracers and meters that will send their captured data to the Elastic APM server. More details on how to use OpenTelemetry to instrument your app can be found in the [OpenTelemetry.io Swift manual instrumentation docs](https://opentelemetry.io/docs/instrumentation/swift/manual/)
+
+Examples can be found in https://github.com/open-telemetry/opentelemetry-swift/tree/main/Examples[opentelemetry-swift/examples]
+
 [discrete]
 [[additional-components]]
 === Additional components
 APM Agents work in conjunction with the {apm-guide-ref}/index.html[APM Server], {ref}/index.html[{es}], and {kibana-ref}/index.html[{kib}].
 The {apm-guide-ref}/index.html[APM Guide] provides details on how these components work together,
 and provides a matrix outlining {apm-guide-ref}/agent-server-compatibility.html[Agent and Server compatibility].
+
+=== Open Telemetry Components
```

Comment:
```suggestion
[discrete]
=== Open Telemetry Components
```

<img width="1239" alt="Screen Shot 2022-10-30 at 3 14 51 PM" src="https://user-images.githubusercontent.com/5618806/198904108-bc95ced0-b500-411a-a02f-c3d9592f2d00.png">

Did you mean to add this to a new page? If not, you can add the discrete tag to keep it on the "Introduction" page.

Reply:
gotcha!